meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/linux/files/r8a7795-h3ulcb-4x2g-kf-hdmi-doma.dts

```diff
@@ -0,0 +1,36 @@
+/dts-v1/;
+
+#include "r8a7795-h3ulcb-4x2g-kf-doma.dtsi"
+
+/ {
+	rvgc {
+		displays {
+			display-0 {
+				display-map = <0x3>;
+				layers {
+					primary {
+						layer-map = <0x3>;
+						size-w = <1280>;
+						size-h = <720>;
+						pos-x = <0x0>;
+						pos-y = <0x0>;
+						pos-override;
```

Comment:
I thought this should remain as it was before (including resolution):
display-map = <0x1>;

Reply:
fixed